### DISCUSSION TITLE
5.0.2.0

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#267] [Announcements]
(Beta 2)
- Implement display config change detection
- Some bug/regressions fixed

<hr /><em>This discussion was created from the release <a href='https://github.com/mgth/LittleBigMouse/releases/tag/'>5.0.2.0</a>.</em>

**Comment by QueTwelve** [comment]
It won't open after I install it, has this happened to anyone else? if so how did you fix it?

**Reply by Spiritreader** [reply]
if it doesn't open after installation, make sure you did the following:

- Ensure that you uninstalled LittleBigMouse v4. If you ran the installer for v5 while still having the old version installed, uninstall v5 and delete the folder from your ProgramFiles directory. "Upgrading" does not remove some old v4 components as of now.
- Open regedit and delete `\HKEY_CURRENT_USER\Software\Mgth\LittleBigMouse\`
- Reinstall LittleBigMouse v5

You'll have to re-do your monitor settings, but the UI should show up from this point.

**Reply by Meta-Cake** [reply]
I'm having the same issue. I did everything you listed. It still isn't opening the UI, the application is running in the background though. I am able to get 5.0.1.0 to open the UI and run, but 5.0.2.0 fixes the issue with windows breaking when moving to another window, pretty much a necessity

**Reply by mgth** [reply]
you should have a mouse icon in the notification bar maybe it's in the hidden part you can open with the up arrow.

**Reply by Meta-Cake** [reply]
I've clicked on it, that's how I always opened the UI in previous versions. Rolling back to 5.0.1 works.

**Reply by A7243618X** [reply]
I encountered the same issue with 5.0.2, it was first installaion of this app. Version 5.0.1 is working ok-ish. I also tried 4.2.7124.42685 and it show icon in notification bar for only 2seconds but that may be diffrent problem.

**Comment by jeremy-feng** [comment]
I have 5 monitors but it shows nothing.

Is there any step that is necessary after installing it? If so, what should I do? Thanks!

![image](https://github.com/mgth/LittleBigMouse/assets/44312563/5283f5fe-38f9-41aa-a5ec-511a03ffc32c)

**Reply by QueTwelve** [reply]
After downloading the software did you have to do anything else to get it to open? or did it just open.

**Reply by jeremy-feng** [reply]
After the installation is complete, the option to open it is automatically checked.

**Reply by mgth** [reply]
Nothing more, you should see the monitors, can you please click on the gear button and post here the copied configuration ?

**Reply by jeremy-feng** [reply]
This is the copied configuration.Thanks for investigating it!

```
{
  "Layout": {
    "Id": "",
    "AutoUpdate": false,
    "PrimarySource": null,
    "PhysicalBounds": {
      "X": 0.0,
      "Y": 0.0,
      "Width": 0.0,
      "Height": 0.0,
      "Position": {
        "X": 0.0,
        "Y": 0.0
      },
      "Size": {
        "AspectRatio": "NaN",
        "Width": 0.0,
        "Height": 0.0
      },
      "Right": 0.0,
      "Bottom": 0.0,
      "Left": 0.0,
      "Top": 0.0,
      "TopLeft": {
        "X": 0.0,
        "Y": 0.0
      },
      "TopRight": {
        "X": 0.0,
        "Y": 0.0
      },
      "BottomLeft": {
        "X": 0.0,
        "Y": 0.0
      },
      "BottomRight": {
        "X": 0.0,
        "Y": 0.0
      },
      "Center": {
        "X": 0.0,
        "Y": 0.0
      }
    },
    "Enabled": false,
    "LoadAtStartup": true,
    "Priority": "Normal",
    "LoopAllowed": true,
    "LoopX": false,
    "LoopY": true,
    "IsUnaryRatio": false,
    "AdjustPointerAllowed": false,
    "AdjustPointer": false,
    "AdjustSpeedAllowed": false,
    "AdjustSpeed": false,
    "HomeCinema": false,
    "Pinned": false,
    "ConfigLocation": {
      "X": 0.0,
      "Y": 0.0,
      "Width": 0.0,
      "Height": 0.0,
      "Position": {
        "X": 0.0,
        "Y": 0.0
      },
      "Size": {
        "AspectRatio": "NaN",
        "Width": 0.0,
        "Height": 0.0
      },
      "Right": 0.0,
      "Bottom": 0.0,
      "Left": 0.0,
      "Top": 0.0,
      "TopLeft": {
        "X": 0.0,
        "Y": 0.0
      },
      "TopRight": {
        "X": 0.0,
        "Y": 0.0
      },
      "BottomLeft": {
        "X": 0.0,
        "Y": 0.0
      },
      "BottomRight": {
        "X": 0.0,
        "Y": 0.0
      },
      "Center": {
        "X": 0.0,
        "Y": 0.0
      }
    },
    "AllowOverlaps": false,
    "AllowDiscontinuity": false,
    "Algorithm": "Cross",
    "MaxEffectiveDpiX": 0.0,
    "MaxEffectiveDpiY": 0.0
  },
  "Devices": {
    "DeviceName": null,
    "DeviceString": null,
    "DeviceId": "ROOT",
    "DeviceKey": null,
    "DisplayModes": {
      "Count": 0,
      "CountChanged": {},
      "Items": []
    },
    "CurrentMode": {
      "Position": {
        "X": 0.0,
        "Y": 0.0
      },
      "BitsPerPixel": 32,
      "Pels": {
        "AspectRatio": 1.7777777777777777,
        "Width": 1920.0,
        "Height": 1080.0
      },
      "DisplayFlags": 0,
      "DisplayFrequency": 60,
      "DisplayFixedOutput": 0,
      "DisplayOrientation": 0
    },
    "Capabilities": {
      "Size": {
        "AspectRatio": 1.7804054054054055,
        "Width": 527.0,
        "Height": 296.0
      },
      "Resolution": {
        "AspectRatio": 1.7777777777777777,
        "Width": 1920.0,
        "Height": 1080.0
      },
      "LogPixels": {
        "AspectRatio": 1.0,
        "Width": 120.0,
        "Height": 120.0
      },
      "Aspect": {
        "AspectRatio": 1.0,
        "Width": 36.0,
        "Height": 36.0
      },
      "BitsPixel": 32
    },
    "State": {
      "AttachedToDesktop": false,
      "MultiDriver": false,
      "PrimaryDevice": false,
      "MirroringDriver": false,
      "VgaCompatible": false,
      "Removable": false,
      "ModesPruned": false,
      "Remote": false,
      "Disconnect": false
    }
  },
  "Zones": {
    "AdjustPointer": false,
    "AdjustSpeed": false,
    "LoopX": false,
    "LoopY": true,
    "Priority": "Normal",
    "Algorithm": "Cross",
    "Zones": [],
    "MainZones": []
  }
}
```

**Comment by normancates** [comment]
Soooo..... When I tried to download the Windows executable, Windows 10 pounced on it and says its a virus...

I totally expect this is a false positive, but its REALLY annoying to try to work around the virus protection

**Reply by normancates** [reply]
I notice that the v5.x releases are abourt 18MB each. Where the previous ones were no more than 2MB.

Is that expected with the new GUI?

**Reply by mgth** [reply]
yes, the size comes from avalonia dlls that where not needed with wpf.

**Reply by mgth** [reply]
About the virus detection, I dont know what to do, I used to compile setup with inno but microsoft often detect it as a virus so I switch back to nullsoft, because VirusTotal would not complain about it. But it may have changed. Microsoft is very annoying about that.
